blob/blob.go

```diff
@@ -13,7 +13,8 @@
 // limitations under the License.
 
 // Package blob provides an easy and portable way to interact with blobs
-// within a storage location, hereafter called a "bucket".
+// within a storage location, hereafter called a "bucket". See
+// https://gocloud.dev/howto/blob/ for getting started guides.
```

Comment:
"for getting started guides" reads a bit weird, like it's missing punctuation. Maybe just "for a tutorial"?

Reply:
Would "how-to guides" read better? The tutorials are a different section.

Reply:
It would read better, though I'm not sure I would understand the distinction between "tutorials" and "how-to guides". It's a bit too fine-grained in my mind. I think of documentation in terms of "tutorial" vs "reference". Not feeling strongly about this though - wording suggestion LGTM

Reply:
Done.